src/nni_manager/training_service/test/localTrainingService.test.ts

```diff
@@ -72,6 +72,31 @@ describe('Unit Test for LocalTrainingService', () => {
         chai.expect(jobDetail.status).to.be.equals('USER_CANCELED');
     }).timeout(20000);
 
+    it('Get trial log', async () => {
+        // set meta data
+        const trialConfig: string = `{\"command\":\"python3 mockedTrial.py\", \"codeDir\":\"${localCodeDir}\",\"gpuNum\":0}`
+        await localTrainingService.setClusterMetadata(TrialConfigMetadataKey.TRIAL_CONFIG, trialConfig);
+
+        // submit job
+        const form: TrialJobApplicationForm = {
+            sequenceId: 0,
+            hyperParameters: {
+                value: 'mock hyperparameters',
+                index: 0
+            }
+        };
+        const jobDetail: TrialJobDetail = await localTrainingService.submitTrialJob(form);
+
+        chai.expect(jobDetail.status).to.be.equals('WAITING');
+        localTrainingService.run()
+        await delay(5000);
+        //chai.expect(jobDetail.status).to.be.equals('RUNNING');
+        chai.expect(await localTrainingService.getTrialLog(jobDetail.id, 'TRIAL_LOG')).to.be.a('string');
```

Comment:
why not compare with 'This is trial log'?

Reply:
This part need refactor, since the submitted jobs are not actually "running". The `run` function doesn't take effect. After sync with liuzhe, it may be better to add this test in IT. Maybe I should arrange an meeting to discuss issues about this pr.

Reply:
in line 91, you already run the training service, why it does not take effect? @liuzhe-lz @chicm-ms

Reply:
I add a `generate_logfile` function in mockedTrial.py to generate mock logfiles. However, these files seems not generated after `run()`.

Reply:
why?